### PR TITLE
Feat(back) : full factory and appfixture

### PR DESCRIPTION
**Pull request by justinDev91**
# Ticket Notion
Full entity factory and appfixtures

# Screen (si besoin)

# Autre information

### CODE REVIEW

**Review by PatRider64 — APPROVED** [review]

File: src/src/Controller/TagControlleeController.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+use App\Entity\Tag;
+use App\Form\TagType;
+use App\Repository\TagRepository;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+#[Route('/tag/controllee')]
```
Comment (MisterAzix) [inline src/src/Controller/TagControlleeController.php:13]:
```suggestion
#[Route('/tag')]
```

File: src/src/Controller/TagControlleeController.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+class TagControlleeController extends AbstractController
```
Comment (MisterAzix) [inline src/src/Controller/TagControlleeController.php:14]:
```suggestion
class TagController extends AbstractController
```

**Review by MisterAzix — CHANGES_REQUESTED** [review]
Good, but change all `tag_controllee` by `tag`

File: src/src/Controller/TagControlleeController.php (hunk 1 above)
Comment (justinDev91) [reply src/src/Controller/TagControlleeController.php:13]: Bien vu

**Review by MisterAzix — APPROVED** [review]

**Pull request merged**